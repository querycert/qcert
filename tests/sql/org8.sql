SELECT name
FROM employees
WHERE age = 32;
