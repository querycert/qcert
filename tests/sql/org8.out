[{"name": "John Doe"}, {"name": "Jane Doe"}, {"name": "Jill Does"}]